frontend/src/components/ListItemCheckbox/ListItemCheckbox.tsx

```diff
@@ -20,6 +20,7 @@ export const ListItemCheckbox: React.FC<Props> = ({ label, subLabel, disabled, c
           checked={checked}
           inputRef={inputRef}
           onChange={event => onClick(event.target.checked)}
+          onClick={event => event.stopPropagation()}
```

Comment:
Checkbox fix.

Reply:
could you add the same for `Select All`?

Reply:
Yes the Select All checkbox uses this component as well, so it should be fixed too.